### ISSUE TITLE
printToSprite  query

### ISSUE THREAD

**Issue by NYFB8888** [Bodmer/TFT_eSPI#3094]
I want to use the printToSprite function, but it doesn't work as I would like.
I used the following code  to test the function 

`for (int i = 0; i <= 10; i++) {`
     ` tft.setCursor(0, 0);`
      `spr.setTextColor(TFT_WHITE, TFT_BLUE);`
     ` //spr.printToSprite(String(i)); //shows nothing but blue background`
     ` //spr.printToSprite(" "+String(i));          //A) shows only 1 and 4 on a blue background `
     ` //spr.printToSprite("0"+String(i));         //B) shows 01 02 etc. on a blue background` 
      `spr.printToSprite(" "+String(i)+" ");       //C) shows 1 2 3 with spaces after and before digit`
      `delay(500);`
   ` }`
Question is possible to print a digit without any space prefix and suffix?

2. IDE both
3. TFT_eSPI library version the latest version
4. Board package version 2.0.11
5. Procesor, ESP32 ESP32-S3
6. TFT driver  ILI9341 and ILI9486
7. Interface type SPI

**Comment by Bodmer** [comment]
No, this is due to the way the print stream sends the string character by character.

Instead use setTextDatum, setTextPadding and drawString functions. These are used in some of the examples.
